moveit_ros/moveit_servo/src/servo_server.cpp

```diff
@@ -52,7 +52,7 @@ int main(int argc, char** argv)
   ros::AsyncSpinner spinner(ROS_THREADS);
   spinner.start();
 
-  ros::NodeHandle nh("~");
+  ros::NodeHandle nh(ros::this_node::getName());
```

Comment:
@tylerjw: I don't understand this change. Aren't private `NodeHandle`s supposed to access the namespace `ros::this_node::getName()`?

Reply:
The private namespace (in my testing) doesn't change with a launchfile name change, instead, it uses whatever string you call init with.

This causes problems with reading parameters because it always looks in /servo_sever/ for some parameters.

Reply:
That would break just about every use of private `NodeHandle`s. I'm not saying you didn't observe what you observed, but I would double check it.

Private `NodeHandle`s are by definition private, in the sense they access parameters *only* from within the `/$node_name` namespace.